Add link to Lambda Days talk by José (#242)
elixir-nx/nx: nx/README.md

  * [The ThinkingElixir podcast where José Valim unveiled Nx](https://thinkingelixir.com/podcast-episodes/034-jose-valim-reveals-project-nx/) (audio)

>   * [The Lambda Days 2021 talk where José shows how to build a neural network with Nx](https://www.youtube.com/watch?v=fPKMmJpAGWc)

Reviewer: ```suggestion
  * [A talk by José Valim at Lambda Days 2021 where he builds a neural network from scratch with Nx](https://www.youtube.com/watch?v=fPKMmJpAGWc)
```
